pywt/_extensions/c/wavelets.h

```diff
@@ -13,6 +13,24 @@ typedef enum {
     SYMMETRIC = 2
 } SYMMETRY;
 
+/* Wavelet name */
+typedef enum {
```

Comment:
I like this idea. I don't think there's a need to assign numbers, `enum {HAAR, REVERSE_BIORTHOGONAL, ...}` should be fine.